testManager.py

```diff
@@ -0,0 +1,92 @@
+from tests import testRequests
+import aiohttp
+import asyncio
+import json
+import API
+
+
+suiteID = 0
+
+async def sendMessage(ws, msg, test=False, testType=None):
+    if test:
+        testFinished = {
+            "message-type": "TEST-FINISHED",
+            "suit-id": suiteID,
+            "test": testType,
+            "results": msg
+        }
+        return await API.sendMessage(ws, json.dumps(testFinished))
+    await API.sendMessage(ws, msg)
+
+
+async def runTests(ws, msg):
+    data = json.loads(msg)
+    testList = list(data['tests'].keys())
+    testConfigs = data['tests']
+    testUrl = data['url']
+    
+    testSuite = await initSuite()
+
+    await sendMessage(ws, createSuite(testList))
+    await sendMessage(ws, startSuite(testList))    
+    await runSuite(ws, testSuite, testConfigs, testUrl)
+
+def createSuite(testList):
+    global suiteID
+    suiteID += 1
+    suiteCreated = {
+        "message-type": "SUITE-CREATED",
+        "suit-id": suiteID,
+        "tests": testList
+    }
+    return json.dumps(suiteCreated)
+
+
+def startSuite(testList):
+    suiteStarted = {
+        "message-type": "SUITE-STARTED",
+        "suit-id": suiteID,
+        "tests": testList
+    }
+    return json.dumps(suiteStarted)
+
+async def initSuite():
+    #Initialize test name and function map
+    testSuite = {
+        #Enter your test functions and names here
+        'testTest': testRequests.testTest,
+        'testTestDuplicate': testRequests.testTestDuplicate
+    }
+
+    return testSuite
+
+#Send singular request with existing asynchronous session
+async def sendRequest(session, url):
+    try:
+        #Fetch individual request content
+        async with session.get(url) as response:
+            if response.status != 200:
+                raise aiohttp.ClientResponseError()
+
+            #Return awaited response content
+            return response
+
+    except Exception as e:
+        return 'Request Failed: ' + 'Error code ' + response.status
```

Comment:
This return type ok? In the in the 'testTest' and 'testTestDuplicate' they check the responses url which will throw an exception and this string will be lost completely. Additionally, the response.status is an int which can not be concatenated with a str